app/views/practices/show/authenticated/_show_authenticated.html.erb

```diff
@@ -1,5 +1,8 @@
 <% provide :head_tags do %>
   <%= javascript_include_tag 'practice_page', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
```

Comment:
⚠️ Only want to track the "Practice show" when the user is logged in so only included in the authenticated view